packages/core/src/lib/implementation/execute-plugin.unit-test.ts

```diff
@@ -144,8 +149,13 @@ describe('executePlugins', () => {
     ).rejects.toThrow(
       'Plugins failed: 2 errors: plugin 1 error, plugin 3 error',
     );
-    expect(console.error).toHaveBeenCalledWith('plugin 1 error');
-    expect(console.error).toHaveBeenCalledWith('plugin 3 error');
+    const logs = ui()
+      .logger.getRenderer()
+      .getLogs()
+      .map(({ message }) => message);
+    expect(logs[0]).toBe('[ yellow(warn) ] Plugins failed: ');
```

Comment:
Is it necessary to have the formatting in this test? Instead, testing various format of the logger should be located in the logger tests.

Applicable to the two lines below as well.

```suggestion
    expect(logs[0]).toEqual(expect.stringContaining('Plugins failed:'));
```

Reply:
I wanted to make sure it is a warning and not a info message so I kept the type of log in all checks so far.

We could in the future check the log type separately as their is additional information available for every log.

Reply:
I opened #552 and move this refactoring there.